profiles/admin.py

```diff
@@ -19,3 +19,9 @@ class ProfileAdmin(VersionAdmin):
     search_fields = ("user__username", "github_account", "user__email", "email")
     list_display = ("github_account", "email", username, user_email)
```

Comment:
(unrelated to your PR changes) but I never noticed this before for how both username and user_email are handled. 🤔